src/main/java/pages/SearchPage.java

```diff
@@ -9,8 +9,8 @@
 @Getter
 public class SearchPage extends PageObject {
 
-    //TODO: add required url for searched product
-    private String pageUrl = "";
+    //TODO: add URL for searched product
+    private String pageUrl = "https://demo.nopcommerce.com/";
```

Comment:
why would you keep TODO comment if you added already?

Reply:
done